src/govuk/common.mjs

```diff
@@ -185,3 +186,16 @@ export function normaliseDataset (dataset) {
 
   return out
 }
+
+/**
+ * Returns the value of the `lang` attribute closest to the given element (including itself)
```

Comment:
This needs to be generalised as this component can be used to get any attribute, not just `lang`.

Reply:
Whoops, started with it only supporting `lang` and forgot to update 😬